### PR TITLE
Update Changelog and remove breaking changes individual file.

### PR DESCRIPTION
**Pull request by katmsft**

### CODE REVIEW

File: sdk/storage/azure-storage-files-datalake/CHANGELOG.md [hunk 1]
```diff
@@ -10,16 +12,27 @@
 * `ETag` and `LastModified` is now `std::string` instead of `Azure::Core::Nullable<std::string>` in `CreateDirectoryResult`, `CreateFileResult` and `CreatePathResult`.
 * `PathAppendDataResult` now returns `ContentMD5`, `ContentCrc64` and `IsServerEncrypted`.
 
+### Breaking Changes
+
+* CreateFromConnectionString now accepts unencoded file, path and directory name.
```
Comment (ahsonkhan) [inline sdk/storage/azure-storage-files-datalake/CHANGELOG.md:17]:
```suggestion
* `CreateFromConnectionString` now accepts unencoded file, path and directory name.
```

File: sdk/storage/azure-storage-files-shares/CHANGELOG.md [hunk 2]
```diff
@@ -14,13 +16,24 @@
   - ShareClient::BreakLease
   - ShareClient::RenewLease
 
+### Breaking Changes
+
+* CreateFromConnectionString now accepts unencoded file and directory name.
```
Comment (ahsonkhan) [inline sdk/storage/azure-storage-files-shares/CHANGELOG.md:21]:
```suggestion
* `CreateFromConnectionString` now accepts unencoded file and directory name.
```

File: sdk/storage/azure-storage-files-shares/CHANGELOG.md [hunk 3]
```diff
@@ -14,13 +16,24 @@
...
+* Added support for getting range list with previous snapshot. `GetFileRangeListResult` now returns `std::vector<FileRange> Ranges` and `std::vector<FileRange> ClearRanges` instead of `std::vector<Range> RangeList`.
+* Added support for SMB Multi-Channel setting for `ServiceClient::GetProperties` and `ServiceClient::SetProperties`. This is only available for Storage account with Premium File access.
+  - Standard account user has to remove the returned SMB Multi-Channel setting before set, otherwise service would return failure.
```
Comment (ahsonkhan) [inline sdk/storage/azure-storage-files-shares/CHANGELOG.md:24]: How is this a breaking change?

**Review by ahsonkhan — APPROVED** [review]
Nice!

File: sdk/storage/azure-storage-files-shares/CHANGELOG.md (hunk 3 above)
Comment (katmsft) [reply sdk/storage/azure-storage-files-shares/CHANGELOG.md:24]: This is a behavioral breaking change. Some code used to work will not be working anymore after this change.
Imagine the scenario where a customer retrieves service properties and set it back without any modification. It used to work but not anymore.

**Review by Jinming-Hu — APPROVED** [review]

**Pull request merged**